README.md

```diff
@@ -1,110 +1,124 @@
 
-## Examples
+## Comparative Examples
 
+[![Visual comparison of WCAG 2.x and APCA](examples/screenshot1.png)](https://xi.github.io/apca-introduction/examples/)
+[![Visual comparison of WCAG 2.x and APCA](examples/screenshot2.png)](https://xi.github.io/apca-introduction/examples/)
 [![Visual comparison of WCAG 2.x and APCA](examples/screenshot.png)](https://xi.github.io/apca-introduction/examples/)
 
+### Official Comparison from Myndex Research:
+This comparison uses achromatic grays as those define readability.
+<img width="540" alt="Visual comparison of WCAG 2.x and APCA" src="https://git.apcacontrast.com/images/ColumnCompareAll400.png">
+
 ## Status
 
-WCAG is an important standard that is a normative part of many laws all over
-the world. If APCA will be part of WCAG 3.0 it will have a huge impact.
-However, currently both WCAG 3.0 and APCA are still in early development.
-Neither is officially recommended by the W3C yet.
+WCAG&nbsp;2 is a guideline developed and recommended by the W3.org. While WCAG&nbsp;2 is not legally binding itself, it has become a normative part of some laws or regulations in some specific jurisdictions around the globe. 
+
+Currently both APCA and the future WCAG&nbsp;3.0 are in active development. At the time of this writing, neither is officially recommended by the W3C.
+
+APCA is also being developed independently for other standards and guidelines,
+
+## Evaluating APCA
+
+Evaluating a contrast algorithm is fairly straight forward as far as determining perceptual uniformity, or in other words, efficacy in terms of the predicted, relative amount of contrast across the visual range. This is because as all sighted users have similarly _shaped_ contrast sensitivity curves when it comes to achromatic luminance, and luminance is the key for readability.
+
+On the other hand, the _thresholds_ chosen for guidelines—i.e., how much contrast does a given element need—is more complicated. All sighted users have substantial contrast needs for best readability. Some users have impairments that cause a reduced contrast sensitivity, and those users need higher contrasts in general. But other users may be over sensitive to high contrasts, and themselves then need lower contrasts overall. 
+
+## More
 
-Evaluating a contrast algorithm is extremly difficult because contrast
-perception varies from person to person and also depends on the lighing
-conditions. Whether APCA is actually better than WCAG 2.x is therefore hard to
-tell. I personally could not say from the examples above which one works better
-for me. A rigorous scientific evaluation is not yet available
-([issue](https://github.com/w3c/silver/issues/574)).
+For a deeper dive, start with [_"Why APCA"_](https://git.apcacontrast.com/documentation/WhyAPCA) and the documentation [_README_](https://git.apcacontrast.com/documentation/README) and the more in depth [link catalog.](https://git.myndex.com) 
```

Comment:
There is no need to add an extra underline to a link

Reply:
` _ ` is NOT an underline, it is italics in GitHub flavored markdown.